RELEASE.rst

```diff
@@ -0,0 +1,13 @@
+RELEASE_TYPE: minor
+
+This release deprecates the ``average_size`` argument to
+:func:`~hypothesis.strategies.lists` and other collection strategies.
+
+The ``average_size`` argument was treated as a hint about the distribution
+of examples from a strategy.  In turn, this is a remnant of earlier versions
```

Comment:
Nit: I'm not sure "In turn" makes sense in this sentence. Certainly it reads oddly to me.

Reply:
(Sorry, that was meant to be the start of a review but I accidentally a single comment. More to follow probably)